functest/build_test.go

```diff
@@ -134,10 +134,9 @@ func TestBuildLabels(t *testing.T) {
 	}
 
 	// create a temporary dir to create the project and run the test
-	projectDir, err := ioutil.TempDir("", "appsody-build-labels-test")
-	if err != nil {
-		t.Fatal(err)
-	}
+	projectDir := cmdtest.GetTempProjectDir(t)
```

Comment:
I was also thinking if you would want to name the TempDir for debugging purposes?

Reply:
Might also be nicer to return a `cleanup` function which would be equivalent to the `os.RemoveAll` passed to `defer`

Reply:
I agree with these comments, however `GetTempProjectDir()` has been used all over in the tests for quite some time now, so I'd like to get this fix in and address these comments in a separate PR.

Reply:
Sounds good!